api/src/main/java/ai/djl/ndarray/NDArray.java

```diff
@@ -3393,6 +3393,51 @@ NDArray stft(
             boolean normalize,
             boolean returnComplex);
 
+    /**
+     * Computes the two-dimensional Discrete Fourier Transform.
+     *
+     * @param sizes Sizes of the transformed dimensions of the output. Will be zero-padded or
+     *     trimmed to this size.
+     * @param dims Dimensions over which to compute the 2D-FFT.
+     * @return The truncated or zero-padded input, transformed along the dimensions indicated by
+     *     dims
+     */
+    NDArray fft2(long[] sizes, long[] dims);
```

Comment:
Can you use `axes` instead of `dims` to be more consistent with the other operations in NDArray?

Reply:
Done